js/main.js

```diff
@@ -98,5 +98,9 @@ document.addEventListener("DOMContentLoaded", function(event) {
         eclipseFdnVideos.replace();
       });
 
+      $(document).on("click", ".pagination li a", function(){
```

Comment:
I feel this is something that should be fix in solstice assets. I think this is a problem that all site has.

Reply:
I created the following MR to add the functionality to solstice assets
https://gitlab.eclipse.org/eclipsefdn/it/webdev/solstice-assets/-/merge_requests/316

Reply:
What is the status of this PR?

Reply:
Yes, the updates were made in this PR.